_posts/2018-08-13-automated-code-styling.md

```diff
@@ -0,0 +1,62 @@
+---
+author: Jorge Gueorguiev Garcia
+layout: post
+asset-type: post
+title: "Automated Coding Style (or why not to use it)"
+date: 2018-08-11 08:01:00
+description: Where I describe my issues with tools like Stylecop and Sonarqube for documentation comments.
+image: 
+    src: /assets/custom/img/blog/2018-08-13-automated-code-styling/caffeine-coffe-cup-6347.jpg
+tags: 
+- software processes
+---
+# Intro
+
+Recently I have had the "privilege" of going through some Javadoc on a project that needs to move from Java 7. The linter for javadoc was tightened for Java 8. Of course, the project did fail to build on Java 8. It fail on so many javadocs.
```

Comment:
Is not the first one correct when trying to emphasize the fact that it happened? Maybe I'm getting confused there.

Reply:
The first sentence may not be technically incorrect, not sure myself, but it's very uncommon to see "did fail" instead of "failed" and may break the reader's immersion.